src/Core/Addon/Module/ModuleManagerBuilder.php

```diff
@@ -145,8 +150,14 @@ public function buildRepository()
         return self::$modulesRepository;
     }
 
-    private function __construct()
+    /**
+     * @param bool $isDebug
+     *
+     * @throws \PrestaShopException
```

Comment:
Where? Otherwise, add a comment to explain how it's possible

Reply:
Fixed